app/components/tab_navigation/view.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module TabNavigation
+  class View < GovukComponent::Base
+    attr_reader :items
+
+    def initialize(items:)
+      @items = items
+    end
+
+  private
+
+    def govuk_link_to(body, url, html_options = {}, &_block)
+      html_options[:class] = prepend_css_class("govuk-link", html_options[:class])
+
+      return link_to(url, html_options) { yield } if block_given?
+
+      link_to(body, url, html_options)
+    end
```

Comment:
Should this be a generic helper? I think it applies to lots of our app and not just the `tab_navigation`

Reply:
Not sure this is actually being used because it's a private method. We do use `govuk_link_to` in other components. `govuk_link_to` is a govuk-components helper and it's included so can be assumed to be available.

Reply:
You're right, it's already available so the bit i added is redundant.